src/main/java/Duke.java

```diff
@@ -1,10 +1,58 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
+//public class Duke {
+//    public static void main(String[] args) {
+//        String logo = " ____        _        \n"
+//                + "|  _ \\ _   _| | _____ \n"
+//                + "| | | | | | | |/ / _ \\\n"
+//                + "| |_| | |_| |   <  __/\n"
+//                + "|____/ \\__,_|_|\\_\\___|\n";
+//        System.out.println("Hello from\n" + logo);
+//    }
+//}
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        GetInPut();
+    }
+
+    public static void GetInPut() {
+        String line;
+        int count = 0;
+        String[] data = new String[100];
+        Scanner input = new Scanner(System.in);
+        line = input.toString();
+
+
+        while (!line.equalsIgnoreCase("bye")) {
+            line = input.nextLine();
+            if (line.equalsIgnoreCase("bye")) {
+                System.out.println("Bye. Hope to see you again soon!");
+                break;
+            }
+            if (line.equalsIgnoreCase("list")) {
+                GetList(data, count);
+            }
```

Comment:
Why don't you put the `else` after `{`, in the same line? :thinking:

Reply:
...